addons/website_portal_sale/security/ir.model.access.csv

```diff
@@ -14,3 +14,5 @@ access_product_attribute_portal,product.attribute portal,product.model_product_a
 access_product_attribute_value_portal,product.attribute value portal,product.model_product_attribute_value,base.group_portal,1,0,0,0
 access_product_attribute_price_portal,product.attribute price portal,product.model_product_attribute_price,base.group_portal,1,0,0,0
 access_product_attribute_line_portal,product.attribute line portal,product.model_product_attribute_line,base.group_portal,1,0,0,0
+access_product_product_portal,product.product.portal,product.model_product_product,base.group_portal,1,0,0,0
+access_product_template_portal,product.template.portal,product.model_product_template,base.group_portal,1,0,0,0
```

Comment:
I understand you want to display the product images but you can't give access to portal users to read all the info of your products.

Best is to create a related field on sale.order.line with the product image, to which he has read access.

I'll do this quickly, get it tested and then merge this PR, thanks!